irods/message/__init__.py

```diff
@@ -180,7 +180,7 @@ def ET(xml_type=(), server_version=None):
 
 logger = logging.getLogger(__name__)
 
-IRODS_VERSION = (4, 3, 4, "d")
+IRODS_VERSION = (5, 0, 1, "d")
```

Comment:
@d-w-moore This should probably be removed since we haven't added support for atime to the PRC.

The atime issue is part of the 3.2.0 milestone.

Reply:
Did i introduce that delta?

Reply:
Removed.